critiquebrainz/frontend/__init__.py

```diff
@@ -75,6 +75,10 @@ def create_app(debug=None, config_path=None):
 
     add_robots(app)
 
+    # BookBrainz Database
+    import critiquebrainz.frontend.external.bookbrainz_db as bookbrainz_db 
+    bookbrainz_db.init_db_engine(app.config.get("BB_DATABASE_URI"))
```

Comment:
For now it would be nice to be able to run CB without a bookbrainz database. To do this, we can check if `app.config.get("BB_DATABASE_URI")` is None, and if so don't run init_db_engine.